l10n_eu_oss/tests/test_l10n_eu_oss.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestL10nEuOss(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestL10nEuOss, cls).setUpClass()
+        # MODELS
+        cls.oss_wizard = cls.env["l10n.eu.oss.wizard"]
+        cls.res_country = cls.env["res.country"]
+        cls.account_tax = cls.env["account.tax"]
+        cls.account_fiscal_position = cls.env["account.fiscal.position"]
+        # INSTANCES
+        # Company
+        cls.company_main = cls.env.ref("base.main_company")
+        # Sale Taxes
+        cls.general_tax = cls.env.ref("l10n_generic_coa.1_sale_tax_template")
```

Comment:
Don't introduce this hidden dependency, or it will fall in other environments. Please create manually a new tax for the tests.